src/app/_components/TextInput/TextInput.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+interface TextInputProps {
+  id: string;
```

Comment:
I personally think id and name should be optional, but that's preference it might be a good to require them.